docs/Getting Started/Understanding Permissions/index.md

```diff
@@ -63,11 +70,35 @@ func hmiLevel(_ oldLevel: SDLHMILevel, didChangeToLevel newLevel: SDLHMILevel) {
     }
 }
 ```
+!@
+
+@![android,javaSE,javaEE]
+```java
+Map<FunctionID, OnRPCNotificationListener> onRPCNotificationListenerMap = new HashMap<>();
+onRPCNotificationListenerMap.put(FunctionID.ON_HMI_STATUS, new OnRPCNotificationListener() {
+    @Override
+    public void onNotified(RPCNotification notification) {
+        OnHMIStatus onHMIStatus = (OnHMIStatus) notification;
+        if (onHMIStatus.getHmiLevel() == HMILevel.HMI_FULL && onHMIStatus.getFirstRun()){
+            // first time in HMI Full
+        }
+    }
+});
+builder.setRPCNotificationListeners(onRPCNotificationListenerMap);
+```
+!@
 
 ## Permission Manager
-When your app first connects to the head unit, it will receive an `OnPermissionsChange` notification. This notification contains all RPCs the head unit supports and the `hmiLevel` permissions for each RPC. Use the `SDLManager`'s permission manager to check the current permission status of a specific RPC or group of RPCs. If desired, you may also subscribe to get notifications when the RPC(s) permission status changes. 
+@![iOS]
+When your app first connects to the head unit, it will receive an `OnPermissionsChange` notification. This notification contains all RPCs the head unit supports and the `hmiLevel` permissions for each RPC. Use the `SDLManager`'s permission manager to check the current permission status of a specific RPC or group of RPCs. If desired, you may also subscribe to get notifications when the RPC(s) permission status changes.
+!@
+
+@![android,javaSE,javaEE]
```

Comment:
Just use the android version for the iOS platform as well.

Reply:
removed tags